src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/Metadata/IModelBindingMessageProvider.cs

```diff
@@ -25,6 +25,13 @@ public interface IModelBindingMessageProvider
         Func<string> MissingKeyOrValueAccessor { get; }
 
         /// <summary>
+        /// Error message the model binding system adds when no value is provided for the request body,
+        /// but a value is required.
+        /// </summary>
+        /// <value>Default <see cref="string"/> is "A value for the request body was not provided.".</value>
+        Func<string> MissingRequestBodyRequiredValueAccessor { get; }
```

Comment:
I'm personally indifferent to whether we add a new validation message or not. @dougbu was against doing so, but the reasoning was to do with a breaking change, which @Eilon says isn't a problem here.

Could the two of you decide whether we should have a new validation message? If so, I'll update the text according to Doug's suggestion (`A non-empty request body is required.`).